### ISSUE TITLE
Is it possible to use pre-specified tokenization when generating bert embeddings?

### ISSUE THREAD

**Issue by loganpeng1992** [imgarylai/bert-embedding#17]
Hi. For datasets with gold tokenizations, is it currently possible to use these gold tokenizations and generate BERT embeddings from those tokens (instead of using BERT's in-house tokenizer from a text string)? Thanks.

**Comment by imgarylai** [comment]
Yeah, I think it is possible, but I never experiment it. As long as the format is good for bert, it should be fine. However, I don't know once you convert your gold tokenizations into bert format, does it hurts your performance. (Just a roughly thinking)

**Comment by loganpeng1992** [comment]
I am using the python imported module for the purpose. I am not sure how to input a list of tokens since the documentation only introduces inputting a string of sentences. Thanks.
